### ISSUE TITLE
List Not Recognized Directories

### ISSUE THREAD

**Issue by isFakeAccount** [b3nj5m1n/xdg-ninja#312]
Is there a way to list not supported directories so it is easier to identify and report them?

**Comment by andrewcrook** [comment]
You mean. any .file and .folder left after a scan?

**Comment by isFakeAccount** [comment]
yes. It would make it easier finding unrecognized directories/files.
